docs/dev/dev_environment.md

```diff
@@ -101,9 +101,6 @@ The project features a CLI helper based on [Invoke](https://www.pyinvoke.org/) t
 
 Each command can be executed with `invoke <command>`. All commands support the arguments `--nautobot-ver` and `--python-ver` if you want to manually define the version of Python and Nautobot to use. Each command also has its own help `invoke <command> --help`
 
-!!! note
-    To run the mysql (mariadb) development environment, set the environment variable as such `export NAUTOBOT_USE_MYSQL=1`.
```

Comment:
No longer accurate/relevant?

Reply:
This comes from the template, `NAUTOBOT_USE_MYSQL` is not in the repo anywhere. To use mysql you can `cp invoke.mysql.yml invoke.yml`.